osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -0,0 +1,34 @@
+from django.shortcuts import _get_queryset
+
+
+def get_object_or_none(klass, *args, **kwargs):
+    """
+    Uses get() to return an object, or None if the object
+    does not exist.
+
+    klass may be a Model, Manager, or QuerySet object. All other passed
```

Comment:
What would you change if it where not so? `class` is still a keyword, and `klass` a reasonable substitute, IMHO.